templates/common/_base/files/nm-ignore-sdn.yaml

```diff
@@ -4,5 +4,5 @@ contents:
   inline: |
     # ignore known SDN-managed devices
     [device]
-    match-device=interface-name:br-int;interface-name:br-local;interface-name:br-nexthop,interface-name:ovn-k8s-*,interface-name:k8s-*;interface-name:tun0;interface-name:br0;driver:veth
+    match-device=interface-name:br-int;interface-name:patch-br-int-*,interface-name:br-local;interface-name:br-nexthop,interface-name:ovn-k8s-*,interface-name:k8s-*;interface-name:tun0;interface-name:br0;driver:veth
```

Comment:
Unmanage patch-br-ex-* and possibly also "int" and "ext" (the latter 2 for hybrid overlay but I'm not sure about this at all)

Reply:
I limited my changes to the the problems I was observing. patch-br-ex- is not an issue on boot since br-ex contents are cleared up first.

It would make sense to add all of them anyway so will do.

Reply:
@andreaskaris @trozet any reason not to add `br-ext` and `ext-vlan`?

Reply:
added them.

So at least when I checked the CI logs, I saw that configure-ovs does no longer get held up with `patch-br-int...`.

Reply:
Let's keep an aye on the upgrade jobs to be sure